frb/surveys/catalog_utils.py

```diff
@@ -388,7 +389,85 @@ def convert_mags_to_flux(photometry_table, fluxunits='mJy'):
         fluxtable[mag][uplimit] = fluxtable[mag][uplimit] 
 
     return fluxtable
+
+def remove_duplicates(tab:Table, idcol:str)->Table:
+    """
+    In an astropy table if there are duplicate
```

Comment:
Comment on what is duplicated.  Is it just ID?

Reply:
Add a line clarifying this.